src/services/auth.js

```diff
@@ -0,0 +1,21 @@
+// TODO: Use typedi or similar
+const Account = require("../models/Account");
+
+class AuthService {
+    constructor() {
+
+    }
+
+    async register(username, password) {
+        const account = await Account.create({
+            username: username,
```

Comment:
You can use the smaller syntax 
```js
{
    username,
    password
}
```

Reply:
bump

Reply:
A bit nitpicking, but fixed :stuck_out_tongue: